WordPress/gutenberg
# Create Block: remove deprecated viewModule field

Follow up #59140

## What?

This PR removes the deprecated and no longer used `viewModule` field from the create-block package.

## Why?

`viewScriptModule` is currently used and the `viewModule` field should be safe to remove (See [this comment](https://github.com/WordPress/gutenberg/pull/59140#discussion_r1494168115)).

## Testing Instructions

Run the following to ensure that the generated block.json still has the expected fields.

- `path/to/gutenberg/packages/create-block/index.js test-block-1`
- `path/to/gutenberg/packages/create-block/index.js test-block-2 --template path/to/gutenberg/packages/create-block-tutorial-template`
- `path/to/gutenberg/packages/create-block/index.js test-block-3 --template path/to/gutenberg/packages/create-block-interactive-template`
